src/renderer/account/components/AccountPanel/index.js

```diff
@@ -8,8 +10,33 @@ import authActions from 'login/actions/authActions';
 import withInitialCall from 'shared/hocs/withInitialCall';
 import withNetworkData from 'shared/hocs/withNetworkData';
 
+import { ID as currentNetworkActionsID } from 'settings/actions/currentNetworkActions';
+import { getStorage } from 'shared/lib/storage';
+import { PREDEFINED_NETWORKS } from 'settings/values/networks';
+
 import AccountPanel from './AccountPanel';
 
+const MAINNET = PREDEFINED_NETWORKS[0];
+
+const tokenListUrl = 'https://raw.githubusercontent.com/CityOfZion/neo-tokens/master/tokenList.json';
+
+const tokenListPromise = (async () => {
+  const response = await fetch(tokenListUrl);
+  const tokenList = Object.values(await response.json());
+  return sortBy(tokenList, 'symbol').map((token) => {
+    return { scriptHash: token.networks['1'].hash };
+  });
+})();
+
+const tokensPromise = (async () => {
+  const currentNetwork = await getStorage(currentNetworkActionsID);
+  if (currentNetwork !== MAINNET) {
+    return [];
+  }
+
+  return tokenListPromise;
+})();
```

Comment:
I've thought about this a bit more, and I'm concerned that if this call fails for any reason, then the entire app will die.  That's not a very good user experience.

With that in mind, I think it would be better for us to encapsulate this into a new redux/spunky action.  We would just need to ensure that call succeeds before we ever try to fetch token balance.

I can help with this if needed.

Reply:
I've added small change so that at least NEO/GAS will still show even if tokenList fetch wasn't successful.